app/models/spree/virtual_gift_card.rb

```diff
@@ -45,7 +47,8 @@ def redeem(redeemer)
       action_originator: self,
       category: store_credit_category,
     )
-    update( redeemed_at: Time.zone.now, redeemer: redeemer )
+
+    update_status
```

Comment:
update_status at this time would be only set as true right? because otherwise an exception was raised

Reply:
Yes, you're right.
I put it cause one of the existing tests was expecting it to return true

Reply:
what do you think would be a better approach?